Module/uti/lightsaber.uti.json

```diff
@@ -129,6 +129,37 @@
           "value": 0
         }
       },
+      {
+        "__struct_id": 0,
```

Comment:
It's always difficult for me to know what's changing in these files. I think you've added some properties to the lightsaber which is probably fine. I just wanted to mention the fact that there is an upgrade process for sabers now. The plan is to enable the player to use them to upgrade their sabers to the next tier instead of fully replacing it. If this has no relevance to the change you made, please disregard.

Reply:
Yes, I added the on hit cast spell power to the lightsaber and saberstaff blueprints.  (I commented that in the commit description but not sure you can easily see that in the review!).

Reply:
I should have checked there, but appreciate the head's up. I'm wondering if we're missing code that applies the OnHit property though? In legacy I was doing that application on equip I believe. Maybe I forgot to carry it over for revamp? I'll add a ticket to review that. It'll be a common thing to forget to add the property so I'd like to automate it.